script/fetch-issues.js

```diff
@@ -14,7 +14,7 @@ let earliestDate = process.env.EARLIEST_DATE || '2017-01-01T00:00:00Z';
 
 if (isDeployPreview) {
   const earliestDateForPr = process.env.EARLIEST_DATE_PR ||
-    '2018-04-01T00:00:00Z';
+    moment().subtract(3, 'months').format('YYYY-MM-DDThh:mm:ssZ');
   if (getTime(earliestDate) < getTime(earliestDateForPr)) {
```

Comment:
I do not see the value in this restriction, especially not now with automatic sensible defaults earliest dates for PRs.

Due to your change, the only way this condition can be false is when an admin of the Netlify control panel sets EARLIEST_DATE_PR to be earlier than EARLIEST_DATE , which is silly.  But if they really want to do that, we should respect that ... maybe they are doing stress testing or something like that.

Reply:
I'm sorry I didn't really get what's to be done here. Can you please brief me a little more?

Reply:
I understand that the `if` condition is always going to be false unless _an admin of the Netlify control panel sets EARLIEST_DATE_PR to be earlier than EARLIEST_DATE_.

But, what am I supposed to do about it in the code?

Reply:
@bhawesh96 Remove this condition; the restriction is not valuable now, especially after your change